corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -5,13 +5,15 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
 
     // Main releases content
     var releasesMainModel = hqImport('app_manager/js/releases/releases').releasesMainModel;
+    var updatePubSub = new ko.subscribable();
```

Comment:
this got more complicated than I expected it would be.

Can't we pass the appReleaseLogModel object created by moving it up, and pass it as a part of the "o" var to releasesMainModel and then directly just call goToPage? or using subscribable the more standard of doing this?

fyi @orangejenny just in case you have quick feedback on this.

Reply:
We can pass appReleaseLogModel object directly to releasesMainModel, but right now it is inside a if block that checks if the table for app release logs was loaded or not and then creates the model, if we move it up it will always be created and then added to releasesMainModel.
Passing data between two viewmodels using `subscribable` was much easier.

Reply:
I don't feel strongly about this. We don't use pubsubs often - web apps is the only other example I'm aware of. So what Manish describes is more typical in our code, yes. That said, I think this is an interesting approach to try.

Reply:
@pxwxnvermx feel free to take whichever path then. My preference is to stick with less code, if it delivers the same value.

Reply:
@mkangia your approach looks fairly straightforward and it will add less code and reduce complexity.